wallet/src/Cardano/Wallet/WalletLayer/Kernel/Settings.hs

```diff
@@ -20,10 +20,15 @@ import           Paths_cardano_wallet (version)
 getNodeSettings :: MonadIO m => Kernel.PassiveWallet -> m V1.NodeSettings
 getNodeSettings w = liftIO $
     V1.NodeSettings
-        <$> (mkSlotDuration <$> Node.getNextEpochSlotDuration node)
+        <$> (V1 <$> Node.getTipSlotId node)
+        <*> (mkSlotDuration <$> Node.getNextEpochSlotDuration node)
+        <*> (V1 <$> Node.getSlotCount node)
         <*> (V1 <$> Node.curSoftwareVersion node)
         <*> pure (V1 version)
         <*> (mkGitRevision <$> Node.compileInfo node)
+        <*> error "TODO"
+        <*> (V1 <$> Node.getFeePolicy node)
+        <*> Node.getSecurityParameter node
   where
```

Comment:
Should this module really exist? Will we not relay the HTTP requests directly to the node API?

Reply:
Oh… *sees this in timeline* https://github.com/input-output-hk/cardano-wallet/commit/fcffff717f61f57a99a1a1ad2f085cc68db6075a 😄